CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# How to contribute to DelugeFirmware/community
+
+Above all, thank you for your interest in both the Synthstrom Deluge and being a contributor!
+
+Please see also, the [Code of Conduct](CODE_OF_CONDUCT.md) document.
+
+## Bug reports
+
+* Please submit an issue highlighting as much detail and context about the bug as possible
```

Comment:
I would propose to phrase it somehow different. 
What's mainly important to me as developer to fix a bug is reproducibility, not necessarily that someone posts all logs and output and whatever might be but probably is not related to the problem. I'm not saying logs are not important, they could help quite a bit to understand - just "as much as possible" sounds a bit too much oriented in the amount and not the quality of the information ;)

A good old tutorial that often helps me to explain this is maybe this one: 
https://www.chiark.greenend.org.uk/~sgtatham/bugs.html

Reply:
@henning strong points all 'round. I'll attempt to reword, and probably include that link as well!